### PR TITLE
[SEO]: fix page not found(external links).

### PR DESCRIPTION
**Pull request by nisarhassan12**
will address the following from #842 :

image

### CODE REVIEW

File: src/blog/build-a-webhook-for-google-assistant.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ Refer to the links below to add your own functionalities.
 
 ## Links to Refer
 * If you would like to learn more about the actions-on-google library, you can find it here - https://www.npmjs.com/package/actions-on-google
-* Go through these examples - https://developers.google.com/actions/samples/github
+* Go through these examples - https://developers.google.com/assistant/conversational/df-asdk/samples/github
```
Comment (gtsiolis) [inline src/blog/build-a-webhook-for-google-assistant.md:137]: Sounds accurate!

**Review by gtsiolis — APPROVED** [review]
Thanks for fixing this @nisarhassan12!

The `client.sh` script shown in the screenshot has moved in https://github.com/Chocobozzz/PeerTube/commit/c026a2e67357bd4bd4fcc4c10f8c5cd4749435c9#diff-f170b9f2b1dbb022fc6aa6f450eb06693d96ab14ddd2711d75585ad4e2bd3751 to `/scripts/dev/client.sh`.

**suggestion**: We can update the link to point to the [new path](https://github.com/Chocobozzz/PeerTube/blob/develop/scripts/dev/client.sh) or even better link using the permalink to a specific commit to avoid introducing such broken links in the future.

/cc @jankeromnes because https://github.com/gitpod-io/website/pull/142

**Comment by ChristinFrohne** [comment]
What is the status here @nisarhassan12 ?

**Comment by nisarhassan12** [comment]
Sorry I have not made the remaining changes yet. Will hopefully make them soon and then merge it.

**Comment by ChristinFrohne** [comment]
Great, thanks :)

**Review by ghuntley — APPROVED** [review]
Merging this as-is now as it resolves a broken link. George your feedback about `client.sh` has been lifted over to https://github.com/gitpod-io/website/issues/842#issuecomment-799868269

**Pull request merged**